code/modules/food&drinks/recipes/tablecraft/recipes_burger.dm

```diff
@@ -177,15 +177,15 @@
 	result = /obj/item/weapon/reagent_containers/food/snacks/burger/purple
 	category = CAT_FOOD
 
-/datum/table_recipe/spellburger
+/datum/crafting_recipe/food/spellburger
 	name = "Spell burger"
 	reqs = list(
 		/obj/item/clothing/head/wizard/fake,
 	)
```

Comment:
should be wizard/fake = 1

This was a bug i fixed, but that was reverted due to other issues.